imagebuildah/build.go

```diff
@@ -284,6 +284,14 @@ func BuildDockerfiles(ctx context.Context, store storage.Store, options define.B
 		}
 
 		builds.Go(func() error {
+			contextDirectory, processLabel, mountLabel, usingContextOverlay, cleanupOverlay, err := platformSetupContextDirectoryOverlay(store, &options)
+			if err != nil {
+				return fmt.Errorf("mounting an overlay over build context directory: %w", err)
+			}
+			defer cleanupOverlay()
+			if contextDirectory != "" {
```

Comment:
(`platformSetupContextDirectoryOverlay` does not document that it might return `""`, this might be made unconditional.)